src/components/Profile.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import {
+  Alert, Col, Container, ListGroup, Row,
+} from 'react-bootstrap';
+import { useSelector } from 'react-redux';
+import '../style/style.scss';
+
+const Profile = () => {
+  const missions = useSelector((state) => state.missions);
+  const joinedMissions = missions.data.slice(1, 7).filter((mission) => mission.reserved);
+  const rockets = useSelector((state) => state.rockets.data);
+  const reservedRocktes = rockets.filter((r) => r.reserved);
+
+  return (
+    <Container className="p-2">
+      <Row md={2}>
+        <Col>
+          <h3>My Missions</h3>
+          {joinedMissions.length ? null : (
+            <Alert>You haven&apos;t joined any missions yet.</Alert>
+          )}
```

Comment:
- [ ] Great work displaying a message if no rockets or missions are reserved. It is truly impressive 👏

Reply:
Thanks, @Gambit142 😊